lib/SILOptimizer/PassManager/PassPipeline.cpp

```diff
@@ -393,6 +393,8 @@ static void addPerfEarlyModulePassPipeline(SILPassPipelinePlan &P) {
   // we do not spend time optimizing them.
   P.addDeadFunctionElimination();
 
+  P.addMandatoryCombiner();
```

Comment:
I am fine with this here, but this may make more sense in the SILOptPreparePassPipeline pipeline. Your thoughts?

Reply:
It happening earlier seems only for the good.  Fixed.